crates/sc-consensus-subspace-rpc/src/lib.rs

```diff
@@ -78,6 +78,15 @@ const SUBSPACE_ERROR: i32 = 9000;
 const SOLUTION_SENDER_CHANNEL_CAPACITY: usize = 9;
 const REWARD_SIGNING_TIMEOUT: Duration = Duration::from_millis(500);
 
+/// The number of object mappings to include in each subscription response message.
+///
+/// This is a tradeoff between `RPC_DEFAULT_MESSAGE_CAPACITY_PER_CONN` and
+/// `RPC_DEFAULT_MAX_RESPONSE_SIZE_MB`. We estimate 500K mappings per segment,
+///  and the minimum hex-encoded mapping size is 88 bytes.
+///
+/// TODO: update the batch size after manual testing.
+const OBJECT_MAPPING_BATCH_SIZE: usize = 10_000;
```

Comment:
I think it is fine to have this as a constant for now, but this is likely a wrong place to have it. RPC parameters can and are changed sometimes with CLI options. So if we reference some default values here, the parameter should probably be a configuration option and either hardcoded as a constant in `subspace-service`/`subspace-node` or better yet derived from other CLI options using some kind of expression.